### PR TITLE
status documentation

### PR DESCRIPTION
**Pull request by alexandrasp**
add a doc for status functionality.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ You should see output like the below in the repl after the 3rd command:
 
 ```
 
+[gRPC Rust Client](gxtdb-rs/docs/gxtdb-rs.md)
```
Comment (naomijub) [inline README.md:111]: `##[gRPC Rust Client](gxtdb-rs/docs/gxtdb-rs.md)` 

Add available endpoints
`[`/status`](https://docs.xtdb.com/clients/clojure/#_status)`

File: gxtdb-rs/docs/gxtdb-rs.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# gXTDB-rs
+
+gXTDB-rs is a gRPC client in Rust for XTDB.
```
Comment (naomijub) [inline gxtdb-rs/docs/gxtdb-rs.md:3]: Rust gRPC client for XTDB

File: gxtdb-rs/docs/gxtdb-rs.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+## gXTDB-rs Client
+
+Once you have called `Client` you will have an instance of the `gRPCClient` struct which has status function to query XTDB via gRPC.
```
Comment (naomijub) [inline gxtdb-rs/docs/gxtdb-rs.md:7]: Code example for creating the `Client`

File: gxtdb-rs/docs/gxtdb-rs.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+
+* [`status`](Doc.rs link)) queries endpoint [`/status`](https://docs.xtdb.com/clients/clojure/#_status) with a `GET`. No args. Returns status ans status response.
```
Comment (naomijub) [inline gxtdb-rs/docs/gxtdb-rs.md:9]: add Link to status example or add example directly

Comment (naomijub) [reply gxtdb-rs/docs/gxtdb-rs.md:9]: Add status example response

File: gxtdb-rs/src/lib.rs [hunk 5]
```diff
@@ -18,11 +18,24 @@ impl From<proto_api::OptionString> for Option<String> {
 }
 
 #[derive(Debug)]
+/// gRPC module written in Rust. It has the gRPC Client for XTDB.
```
Comment (naomijub) [inline gxtdb-rs/src/lib.rs:21]: Not necessary `gRPC module written in Rust.`

File: gxtdb-rs/src/lib.rs [hunk 6]
```diff
@@ -18,11 +18,24 @@ impl From<proto_api::OptionString> for Option<String> {
...
 }
 
 impl Client {
+    /// Implement client needs in order to have a gRPC client instance.
+    ///
+    ///
+    ///  Returns a gRPC client connection.
+    /// # Arguments
+    /// *`host`
+    /// *`port`
```
Comment (naomijub) [inline gxtdb-rs/src/lib.rs:38]: `# Result` section
rustup component add clippy
cargo clippy --all-features -- -W clippy::pedantic --deny "warnings"

File: gxtdb-rs/src/lib.rs [hunk 7]
```diff
@@ -35,7 +48,7 @@ impl Client {
             client: GrpcApiClient::new(channel),
         }
     }
-
+    /// * `status` requests endpoint `/status` via `GET`. No args. Returns status response and status.
```
[the remark this answers is not in the document]
Comment (naomijub) [reply gxtdb-rs/src/lib.rs:51]:
```
# Result
Result<tonic::Response<proto_api::StatusResponse>, tonic::Status>

StatusResponse {
 fields...
}

## Result Json
{}
```

File: src/gxtdb/service.clj [hunk 8]
```diff
@@ -41,6 +41,10 @@
 ;;
 ;; see http://pedestal.io/reference/request-map
 
+;; -- XTDB-DOCS --
+;; Available functionalities:
+;; - submittx: writes transactions in a log and returns details about transactions.
+;; - status: return status of a xtdb-node.
```
Comment (alexandrasp) [inline src/gxtdb/service.clj:47]: @naomijub take a look, could this be useful?

File: gxtdb-rs/docs/gxtdb-rs.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
+# gXTDB-rs
+
+Rust gRPC client for XTDB.
+
+## gXTDB-rs Client
+
+Once you have called `Client` you will have an instance of the `gRPC Client` struct which has status function to query XTDB via gRPC.
```
Comment (naomijub) [inline gxtdb-rs/docs/gxtdb-rs.md:7]: Which has the following functions to query XTDB via gRPC

Comment (naomijub) [reply gxtdb-rs/docs/gxtdb-rs.md:7]: * [`status`](To do: Update to Doc.rs) queries endpoint [`/status`](https://docs.xtdb.com/clients/clojure/#_status) via gRPC. No args. Returns status ans status response.

File: gxtdb-rs/docs/gxtdb-rs.md [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+
+
+### `Client` code example:
+
+```
+let mut client = Client::new("http://localhost", 8080).await.unwrap();
+```
+
+* [`status`](To do: Update to Doc.rs) queries endpoint [`/status`](https://docs.xtdb.com/clients/clojure/#_status) via gRPC. No args. Returns status ans status response.
```
Comment (naomijub) [inline gxtdb-rs/docs/gxtdb-rs.md:16]: Move Up

File: gxtdb-rs/docs/gxtdb-rs.md [hunk 11]
```diff
@@ -0,0 +1,30 @@
...
+
+### Status Example Response:
```
Comment (naomijub) [inline gxtdb-rs/docs/gxtdb-rs.md:18]: ##

**Review by naomijub — APPROVED** [review]

**Pull request merged**